package.json

```
@@ -1,5 +1,8 @@
 {
   "dependencies": {
+    "@chakra-ui/next-js": "^2.4.2",
+    "@chakra-ui/react": "^3.2.3",
+    "@emotion/react": "^11.14.0",
```

```suggestion
    "@chakra-ui/next-js": "^2.4.2",
    "@chakra-ui/react": "^3.2.3",
    "@emotion/react": "^11.14.0",
```

wrong folder, uninstall these libraries from here